pkg/session/bootstrap.go

```diff
@@ -3373,6 +3376,8 @@ func upgradeToVer243(s sessiontypes.Session, ver int64) {
 	}
 	doReentrantDDL(s, "ALTER TABLE mysql.tidb_global_task ADD COLUMN max_node_count INT DEFAULT 0 AFTER `modify_params`;", infoschema.ErrColumnExists)
 	doReentrantDDL(s, "ALTER TABLE mysql.tidb_global_task_history ADD COLUMN max_node_count INT DEFAULT 0 AFTER `modify_params`;", infoschema.ErrColumnExists)
+	doReentrantDDL(s, "ALTER TABLE mysql.tidb_global_task ADD COLUMN extra_params json;", infoschema.ErrColumnExists)
```

Comment:
it's better to use `AFTER column` clause

Reply:
https://github.com/pingcap/tidb/pull/59641/commits/27e3283c095486f3031a370f868b19a3d66634fe